_pages/policies/employee-resources-policies/deaf-hoh.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Resources for Deaf and Hard of Hearing individuals in TTS
+tags:
```

Comment:
@tbaxter-18f these tags!

Reply:
Thanks @jenniferthibault I did a bit of research, and found that _most_ pages aren't using them at all, but the few that are (like `policies/benefits-links.md` for example) appear to be using them keyword stuff related search terms that may not be in the document text. In this case, I can't think of any search terms that would be related to the content that aren't already in the content, so I think you could do as most of the pages are doing and simply drop it altogether.

Reply:
Awesome, thank you!